__tests__/helpers/toRespondWithChunks.ts

```diff
@@ -0,0 +1,57 @@
+import { ClientRequest, IncomingMessage } from "http";
+
+/**
+ * This matcher compares the received chunks with the provided ones.
+ * It wraps the provided values using `Buffer.from` so that there's no need
+ * to do so in the tests. The matcher will exit and close the connection
+ * as soon as the condition match or fail.
+ *
+ * Please note that this is an async matcher thus the tests to be async as well.
+ *
+ * @param {"http".ClientRequest} req
```

Comment:
🤔 Do we still need this type of docstring when we use typescript?

Reply:
I'd discard it

Reply:
Probably not, I'll remove it...